sentry-core/src/main/java/io/sentry/core/protocol/Contexts.java

```diff
@@ -27,6 +27,8 @@ public void setBrowser(Browser browser) {
   }
 
   public Device getDevice() {
+    // getter is not working -> Cannot cast com.google.gson.internal.LinkedTreeMap to
```

Comment:
@bruno-garcia should I change this class with fields? it's gonna solve it

Reply:
it seems that we need to have as a `map` because we need to support any type of object that comes from our users.

> Unknown data for the contexts is rendered as a key/value list.
@bruno-garcia is it because of `BeforeSend` as well?

Reply:
It's a way for users to add their own serializable objects to be displayed in Sentry.
The Contexts interface (like `Extra`) will display your serialized object (JSON) nicely and it's great for debugging

Reply:
let's do in a different PR as well, it's not related to this one

Reply:
agreed